kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -233,18 +235,23 @@ public void flush(Map<TopicPartition, OffsetAndMetadata> offsets) {
       } catch (InterruptedException err) {
         throw new ConnectException("Interrupted while scheduling/executing write threads", err);
       } catch (ExecutionException err) {
-        throw new BigQueryConnectException("Exception occurred while executing write threads", err);
+        throw new BigQueryConnectException("Exception occurred while executing write threads",
+                                           err);
       }
     }
     resumeAllPartitions();
   }
 
-  private TableId getRecordTable(SinkRecord record) {
-    return TopicToTableResolver.getPartitionedTableName(topicsToTableIds.get(record.topic()));
+  private PartitionedTableId getRecordTable(SinkRecord record) {
+    TableId baseTableId = topicsToBaseTableIds.get(record.topic());
+    return new PartitionedTableId.Builder(baseTableId).setDayPartitionForNow().build();
```

Comment:
Regarding `setDayPartitionForNow()`, should it consider the record timestamp instead ? It seems that records are always inserted in the partition for the current day; But I could consume a topic with "old" records, or use a custom timestamp extractor.
@mtagle @criccomini